osu.Game/Overlays/OSD/Toast.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Sprites;
+using osuTK;
+using osuTK.Graphics;
+
+namespace osu.Game.Overlays.OSD
+{
+    public abstract class Toast : Container
+    {
+        private readonly Container content;
+        protected override Container<Drawable> Content => content;
+
+        protected readonly OsuSpriteText ValueText;
```

Comment:
why not `private`?

Reply:
This member is accessed in `TrackedSettingToast` to set the Anchor of the value sprite text in case of a setting with multiples options

![image](https://user-images.githubusercontent.com/20256717/62854586-098e3b80-bcf0-11e9-885d-95a163c7e7a0.png)